app/models/parent.rb

```diff
@@ -25,13 +25,19 @@ class Parent < ApplicationRecord
 
   has_one :patient
 
+  has_and_belongs_to_many :cohort_imports
+
   attr_accessor :parental_responsibility
 
   enum :contact_method, %w[text voice other any], prefix: true
   enum :relationship, %w[mother father guardian other], prefix: true
 
   encrypts :email, :name, :phone, :relationship_other
 
+  normalizes :phone,
+             with: ->(str) { str.blank? ? nil : str.to_s.gsub(/\s/, "") }
+  normalizes :email, with: ->(str) { str.nil? ? nil : str.to_s.downcase.strip }
```

Comment:
TIL https://www.bigbinary.com/blog/rails-7-1-adds-activerecord-base-normalizes

Reply:
Yeah, it's a nice feature!